tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -788,78 +788,6 @@ def test_negative_update_with_name(self, new_name, content_view, module_target_s
                 {'content-view-id': content_view['id'], 'name': new_name}
             )
 
-    @pytest.mark.tier1
-    def test_negative_update_with_same_name(self, module_org, content_view, module_target_sat):
-        """Try to update content view filter using name of already
-        existing entity
-
-        :id: 9c1b1c75-af57-4218-9e2d-e69d74f50e04
-
-        :expectedresults: Content view filter is not updated
-
-        :CaseImportance: Critical
-        """
-        cvf_name = gen_string('utf8')
-        module_target_sat.cli.ContentView.filter.create(
-            {
-                'content-view-id': content_view['id'],
-                'name': cvf_name,
-                'organization-id': module_org.id,
-                'type': 'rpm',
-            },
-        )
-        new_name = gen_string('alpha', 100)
-        module_target_sat.cli.ContentView.filter.create(
-            {
-                'content-view-id': content_view['id'],
-                'name': new_name,
-                'organization-id': module_org.id,
-                'type': 'rpm',
-            },
-        )
-        with pytest.raises(CLIReturnCodeError):
-            module_target_sat.cli.ContentView.filter.update(
-                {
-                    'content-view-id': content_view['id'],
-                    'name': new_name,
-                    'new-name': cvf_name,
-                }
-            )
-
-    @pytest.mark.tier1
-    def test_negative_update_inclusion(self, module_org, content_view, module_target_sat):
```

Comment:
I believe this deletion must be due to low severity of test, don't see any qe-recommendation in the sheet.

Reply:
also listed as delete on sheet